src/Mailer/Email.php

```diff
@@ -827,6 +848,38 @@ public function headerCharset($charset = null)
         return $this->headerCharset;
     }
 
+    /**
+     * TransferEncoding setter.
+     *
+     * @param string|null $encoding Encoding set.
+     * @return $this
+     */
+    public function setTransferEncoding($encoding)
```

Comment:
Should we be handling encoding the message based on the indicated transfer encoding? Right now someone could set the transfer encoding to base64, but the email message will still be in 8bit (generally).

Reply:
I think than that two differents parts. I don't know if have a link between transfer encoding and message encoding.

Reply:
Ok.